Octokit/Clients/RepositoriesClient.cs

```diff
@@ -86,6 +86,11 @@ public Task<Repository> Create(string organizationLogin, NewRepository newReposi
                     errorMessage,
                     StringComparison.OrdinalIgnoreCase))
                 {
+                    if (string.IsNullOrEmpty(organizationLogin))
+                    {
+                        throw new RepositoryExistsException(newRepository.Name, e);
+                    }
```

Comment:
> Ah. And we don't have access to the name because when we do operations for the current account, the name isn't needed as part of the URL. Right?

Exactly :smile: at least the integration test is passing now!